Core/Core/InstUI/Styles/Padding.swift

```diff
@@ -18,27 +18,118 @@
 
 import SwiftUI
 
-public extension InstUI.Styles {
-    enum Padding: CGFloat {
+// MARK: - Values
+
+extension InstUI.Styles {
+
+    public enum Padding: CGFloat {
         case standard = 16
+
         case cellTop = 12
         case cellBottom = 14
+        /// The horizontal padding before a cell's leading icon
+        case cellIconLeading = 22
         /// The horizontal padding between a cell's leading icon and its text
         case cellIconText = 18
-        case cellLeading = 22
+
         case paragraphTop = 24
         case paragraphBottom = 28
+
         /// When displaying multiple Text components below each other we use this spacing to separate them
         case textVertical = 4
+
+        /// Correction to negate baked in TextEditor inset. Estimated value.
+        case textEditorVerticalCorrection = -7
+        /// Correction to negate baked in TextEditor inset. Estimated value.
+        case textEditorHorizontalCorrection = -5
+    }
+
+    public enum PaddingSet {
+        /// Default paddings for cells
+        case standardCell
+        /// Paddings for cells with leading icon
+        case iconCell
+        /// Paddings to negate TextEditor insets
+        case textEditorCorrection
+
+        var config: Config {
+            switch self {
+            case .standardCell:
+                Config(
+                    top: .cellTop,
+                    bottom: .cellBottom,
+                    leading: .standard,
+                    trailing: .standard
+                )
+            case .iconCell:
+                Config(
+                    top: .cellTop,
+                    bottom: .cellBottom,
+                    leading: .cellIconLeading,
+                    trailing: .standard
+                )
+            case .textEditorCorrection:
+                Config(
+                    top: .textEditorVerticalCorrection,
+                    bottom: .textEditorVerticalCorrection,
+                    leading: .textEditorHorizontalCorrection,
+                    trailing: .textEditorHorizontalCorrection
+                )
+            }
+        }
+
+        struct Config {
+            let top: Padding?
+            let bottom: Padding?
+            let leading: Padding?
+            let trailing: Padding?
+        }
     }
 }
 
-public extension View {
+// MARK: - Modifiers
+
+extension View {
 
-    @inlinable func paddingStyle(
+    @inlinable
+    public func paddingStyle(
         _ edges: Edge.Set = .all,
         _ padding: InstUI.Styles.Padding? = nil
     ) -> some View {
         self.padding(edges, padding?.rawValue)
     }
+
+    public func paddingStyle(set: InstUI.Styles.PaddingSet) -> some View {
+        self
+            .applyPadding(edge: .top, padding: set.config.top)
+            .applyPadding(edge: .bottom, padding: set.config.bottom)
+            .applyPadding(edge: .leading, padding: set.config.leading)
+            .applyPadding(edge: .trailing, padding: set.config.trailing)
```

Comment:
Could we apply the padding with the built-in `.padding()` method? It seems that the private `PaddingModifier` and the `applyPadding` function adds no value.